cdc/capture/http_validator_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package capture
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/pingcap/ticdc/cdc/model"
+	"github.com/pingcap/ticdc/pkg/config"
+	"github.com/pingcap/ticdc/pkg/util/testleak"
+)
+
+func TestVerifyUpdateChangefeedConfig(t *testing.T) {
+	defer testleak.AfterTestT(t)()
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	oldInfo := &model.ChangeFeedInfo{Config: config.GetDefaultReplicaConfig()}
+	// test startTs > targetTs
+	changefeedConfig := model.ChangefeedConfig{TargetTS: 20}
+	oldInfo.StartTs = 40
+	newInfo, err := verifyUpdateChangefeedConfig(ctx, changefeedConfig, oldInfo)
+	assert.NotNil(t, err)
```

Comment:
seems in tidb, require is preferred https://github.com/stretchr/testify/#require-package

Reply:
thanks ~